sos/plugins/python.py

```diff
@@ -20,6 +23,9 @@ class Python(Plugin, DebianPlugin, UbuntuPlugin):
     profiles = ('system',)
 
     packages = ('python',)
+    option_list = [
+        ('gather_py_hashes', "gather hashes for all python files", 'slow',
+         False)]
```

Comment:
As written, this will add the `gather_py_hashes` option to the python plugin for all current distributions. However, it will only execute on Red Hat systems. That's not so great.

There are a few options here:
- Pull the `gather_py_hashes` logic out of `setup()` into its own method, which `setup()` then calls, but this would still require duplicate checks in each `*Python` class as currently written.
- Do the above, but have `RedHatPython` call `super(Python, self).setup()` before it does its own collections, and then we'd need to clean up duplicate writes to `python-version`.
- Move the `option_list` definition to `RedHatPython` and have this option only available to Red Hat family systems.

Reply:
Thanks

Reply:
@TurboTurtle Should I choose only one of these options? The easiest for me looks the 3rd option. I will go with that and squash.